src/components/pages/donation_form/DonationSummary.vue

```diff
@@ -1,60 +1,44 @@
 <template>
 	<div class="form-summary-content">
```

Comment:
Because this section changes dynamically it should be given an [aria-live](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Attributes/aria-live) attribute, I would mark it as polite.
```suggestion
	<div class="form-summary-content" aria-live="polite">
```